gr-utils/python/modtool/gr-newmod/python/build_utils.py

```diff
@@ -55,96 +57,95 @@
 
 name_dict = {}
 
-def log_output_name (name):
-    (base, ext) = os.path.splitext (name)
+def log_output_name(name):
```

Comment:
I guess your linter would also complain about the lack of a docstring?

Reply:
Yes, there are a lot of functions with missing docstring. Almost all the functions in `build_utils_codes.py`.